### PR TITLE
Extra lines for NPCs

### PR DESCRIPTION
**Pull request by Chambs-15**
Some new lines for NPCs and removal of some duplicate lines.

### CODE REVIEW

File: data/json/npcs/talk_tags.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
         "category" : "<name_b>",
         "text" : [
             "punk",
+            "wanker",
```
Comment (Leland) [inline data/json/npcs/talk_tags.json]: do note that this game is set stateside, and "wanker" is predominantly British

Comment (RobertoVGoulart) [inline data/json/npcs/talk_tags.json]: Should i remove it or is it used on US enough to be acceptable?

**Comment by kevingranade** [comment]
Wanker is almost never used in the US except by british-speaking expats.

**Comment by RobertoVGoulart** [comment]
Removed "Wanker"

**Pull request merged**